apps/crn-server/test/data-providers/contentful/event.data-provider.test.ts

```diff
@@ -1033,7 +1033,7 @@ describe('Events Contentful Data Provider', () => {
           status: {
             'en-US': 'Confirmed',
           },
-          tags: {
+          researchTags: {
```

Comment:
is there a type for all of this input? if yes then you should consider adding `satisfies ....` at the end of this embedded object fixture